webapp/backend/views.py

```diff
@@ -955,7 +955,18 @@ def destroy(self, request, uuid, format=None):
                                              .format(state="destroyed"))
 
         if lambda_instance_data['status'] == LambdaInstance.CLUSTER_FAILED:
-            raise CustomCantDoError(CustomCantDoError.messages['cant_do'].
+            if 'out of limit' in lambda_instance_data['failure_message']:
+                events.set_lambda_instance_status.delay(lambda_instance_data['uuid'],
```

Comment:
No, it would have to be CLUSTER_FAILED, to be deleted using this method. That was also the intended behavior,  so that a user can remove non-existent instances from the list.